Makefile

```diff
@@ -1,9 +1,10 @@
-dev:
+dev-build:
 	docker build --rm -f Dockerfile.dev -t jb_hugo:latest .
+
+dev: dev-build
 	docker run --rm -it -v "$${PWD}":/app -w /app -p $${HOST_IP:-127.0.0.1}:1313:1313 jb_hugo:latest
 
-dev-build-metrics:
-	docker build --rm -f Dockerfile.dev -t jb_hugo:latest .
+dev-build-metrics: dev-build
```

Comment:
(same thing as above) this tells make, if you're going to do this step then `dev-build` is a dependency of this step.

Reply:
nice!